crates/stackable-webhook/src/tls.rs

```diff
@@ -97,7 +103,10 @@ impl TlsServer {
             .private_key_der()
             .context(EncodePrivateKeyDerSnafu)?;
 
-        let mut config = ServerConfig::builder()
+        let tls_provider = default_provider();
+        let mut config = ServerConfig::builder_with_provider(tls_provider.into())
+            .with_safe_default_protocol_versions()
```

Comment:
This enables both TLS1.2 and TLS1.3. I didn't put much thought into this option.

Alternatively, select the versions via [`with_protocol_versions(...)`](https://docs.rs/rustls/latest/rustls/struct.ConfigBuilder.html#method.with_protocol_versions)

Reply:
Using `with_protocol_versions` is a little more explicit. I'm not 100% sure if we need to be that explicit, or if using the default ones (TLS1.2 and TLS1.3) is fine.

Reply:
Done in c3122c9